### ISSUE TITLE
Blank space on search screen

### ISSUE THREAD

**Issue by shaunan** [StatCan/StatcanPublicationUAT#601]
On Android devices, when navigating the screen after selecting the Cancel button after doing a search, the focus indicator goes on a big blank space. This should be removed. 

![GitHub14](https://github.com/StatCan/StatcanPublicationUAT/assets/115822055/427c91f9-8ebf-4ade-aad5-5ca09071bb69)
